00 - Contribute to the Obsidian Hub/02 Attachments/🗂️ 02 Attachments.md

```diff
@@ -71,11 +71,6 @@ publish: true
 -  [[00 - Contribute to the Obsidian Hub/02 Attachments/VSCode Guide - Search.gif|VSCode Guide - Search.gif]]
 -  [[00 - Contribute to the Obsidian Hub/02 Attachments/VSCode Guide - Toggle Wrapping.gif|VSCode Guide - Toggle Wrapping.gif]]
 -  [[00 - Contribute to the Obsidian Hub/02 Attachments/vscode-regex-find-replace.png|vscode-regex-find-replace.png]]
--  [[00 - Contribute to the Obsidian Hub/02 Attachments/YAFPA_Showcase_Category_view.png|YAFPA_Showcase_Category_view.png]]
```

Comment:
I'm not qualified to comment on the deletion of the YAFPA note....

However, just to note that it looks like the links to the images have been removed, but the images themselves haven't been removed.

As this list is computer-generated, these deletions will get re-added.

So I think the YAFPA images should all be removed.

Reply:
Oh, I completly forgot for the image, sorry ! I will delete them asap.